.buildkite/scripts/dra-workflow.trigger.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I'm not seeing what's actually referencing this script. Is it another external pipeline definition?